content/about/limits.md

```diff
@@ -1,44 +1,78 @@
+
+### Daily Request Limit
+
+Accounts using the Workers *free* tier are subject to a **daily request limit of 100,000 requests**. Free tier daily requests counts reset at midnight UTC. A Worker that fails as a result of daily request limit errors can be configured by setting its corresponding [route](/about/routes/) in two modes: _Fail open_ and _Fail closed_.
 
-| Plan             | CPU  |
-| ---------------- | ---- |
-| Free             | 10ms |
-| Unlimited (Paid) | 50ms |
+#### Fail Open
+
+Routes in fail open mode will bypass the failing Worker and prevent it from operating on incoming traffic. Incoming requests will behave as if there was no Worker.
+
+#### Fail Closed
+
+Routes in fail closed mode will display a Cloudflare 1027 error page to visitors, signifying the Worker has been temporarily disabled. We recommend this option if your Worker is performing security related tasks.
+
+<!-- Add workers request limit error code -->
+
+## CPU/Execution Time Limit
 
-Most Workers requests consume less than a millisecond. It’s rare to find a normally operating Workers script that exceeds the CPU time limit. The 10ms the Free plan allows is enough runtime for most use cases, including application hosting.
+Most Workers requests consume less than a millisecond. It’s rare to find a normally operating Workers script that exceeds the CPU time limit. A Worker may consume up to **10ms on the free tier** and **50ms on the Unlimited tier**. The 10ms allowance on the Free plan is enough execution time for most use cases including application hosting.
 
 There is no limit on the real runtime for a Workers script. As long as the client that sent the request remains connected, the Workers script can continue processing, making subrequests, and setting timeouts on behalf of that request. When the client disconnects, all tasks associated with that client request are canceled. You can use [`event.waitUntil()`](/reference/apis/fetch-event/) to delay cancellation for another 30 seconds or until the promise passed to `waitUntil()` completes.
 
 ## Memory
 
-Only one Workers instance runs on each of the many global Cloudflare edge servers. Each Workers instance can consume up to 128MB of memory. Use [global variables](/reference/apis/standard/) to persist data between requests on individual nodes; note however, that nodes are occasionally evicted from memory.
+Only one Workers instance runs on each of the many global Cloudflare edge servers. Each Workers instance can consume up to **128MB** of memory. Use [global variables](/reference/apis/standard/) to persist data between requests on individual nodes; note however, that nodes are occasionally evicted from memory.
```

Comment:
Perhaps we want to add something here about HTMLRewriter - a common reason to run into this memory limit is because they are reading in large chunks of HTML and when concurrent requests come in their worker starts to churn